source/GameContainer.js

```diff
@@ -14,10 +15,8 @@ export default class GameContainer extends Pixi.Container {
         this.countdownToJunk = Utility.randomNumber(Reference.JUNK_FREQUENCY_RANGE.lower, Reference.JUNK_FREQUENCY_RANGE.upper)
         Textures.initTex()
     }
- 
+
     gameOver() {
```

Comment:
Yep. This will sit until we put in the logic for high score entry and the like. Added a TODO comment to that effect.